misc/package/Data Files/MWSE/core/lib/logger/colors.lua

```diff
@@ -0,0 +1,83 @@
+--[[Logging colors.
```

Comment:
it has been done. i just copypasted the existing `colors.lua` into the new logger folder since i'm still not sure about backwards compatibility (or which folder this logger will sit in once everything is finished)